pages/ParentGuide.js

```diff
@@ -400,22 +400,22 @@ export const ParentGuideInformation = ({ route, navigation }) => {
             >
                 <InformationContainer>
                     <StandardTextbox
```

Comment:
You may want to reduce the weight of the category title to match Figma.

Reply:
This is a design thing, but I feel that INTERPRETATIONS followed by People, Place etc. doesn't make sense to me. I would just have the interpretation (The people your child mention they find trustworthy or whatever). What do others think?

Reply:
I decided to bump the weight up a bit or else everything will get too light.

Reply:
(It looks way lighter than figma's)